design-documents/graph-ql/coverage/store-pickup.graphqls

```diff
@@ -0,0 +1,29 @@
+type Query {
+    pickupLocations (
+        filter: PickupLocationFilterInput,
+        pageSize: Int = 20,
+        currentPage: Int = 1
+    ): PickupLocations
+}
+
+input PickupLocationFilterInput {
+    # This type is added for extensibility
+    search_term: String!
```

Comment:
For me it is not clear how `search_term` could work with current API implementation.
Do I correctly understand that:
- Search Term - search string to search by customer address
- we delegate all logic of parsing entered address to server-side php scripts, instead of client-side javascript
- radius for the search will be taken from configurations and there is no way to pass it without customizations

Reply:
The questions for now:
- it is not clear how correctly parse `search_term` into address. Offline address search does not support 'just a string'
- if `search_term` could include search by attribute values (like name of Pickup Location (PL) or some address fields) it is not clear how to extract this knowledge from search string
- probably, fulltext index could be as solution for PL attributes, but it will not solve issue with Offline search.

Reply:
Thank you, will add `radius` and ability to query radius units, google/offline search mode.